kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -241,6 +241,57 @@
     padding: 8px;
 }
 
+.kb-staging-table-import {
+    position: relative;
+}
+
+.kb-staging-table-import__button {
+    width: 160px;
+    height: 42px;
+    border-radius: 4px;
+    font-family: Public Sans, Arial, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 16px;
+    position: absolute;
+    right: 0;
+    margin-top: 16px;
+}
+
+.kb-staging-table-import__button.kb-staging-table-import__button__disabled,
```

Comment:
does it need the extra `.kb-staging-table-import__button` or is `.kb-staging-table-import__button__disabled` specific enough?

Reply:
Why did you choose to do `button__disabled` instead of `button:disabled`

Reply:
Just kidding I think I see now. I didn't look below before asking

Reply:
@ialarmedalien - i can look at that, probably just disabled is enough.

@eamahanna - it's a good question, sorry that's something i should have commented on. i wound up going with a disabled class rather than actually making the button functionality disabled because otherwise the tooltip wouldn't display

Reply:
ahh nope @ialarmedalien we do need that class to override the default button styles